x-pack/plugins/fleet/server/services/fleet_server/index.ts

```diff
@@ -50,22 +50,25 @@ export async function startFleetServerSetup() {
     _onResolve = resolve;
   });
   const logger = appContextService.getLogger();
+
+  // Check for security
   if (!appContextService.hasSecurity()) {
     // Fleet will not work if security is not enabled
     logger?.warn('Fleet requires the security plugin to be enabled.');
     return;
   }
 
+  // Log information about custom registry URL
+  const customUrl = appContextService.getConfig()?.registryUrl;
```

Comment:
as we no longer require a license check, I moved this out of the `try` block below